packages/now-cli/test/dev/integration.js

```diff
@@ -601,15 +697,14 @@ test('[now dev] 12-polymer-node', async t => {
     // start `now dev` detached in child_process
     dev.unref();
 
-    const result = await fetchWithRetry(`http://localhost:${port}`, 180);
-    const response = await result;
+    const response = await fetchWithRetry(`http://localhost:${port}`, 180);
 
     validateResponseHeaders(t, response);
 
     const body = await response.text();
     t.regex(body, /Polymer \+ Node.js API/gm);
   } finally {
-    dev.kill('SIGTERM');
+    await dev.kill('SIGTERM');
```

Comment:
I noticed some tests use `testFixtureStdio()` and some do not. Why is that?

Reply:
`testFixtureStdio` was introduced later, and it doesn't provide a way to check the `stdio` of the `now dev` process, but some need to do that